oaknational/oak-ai-lesson-assistant
# feat: add additional materials button - AI-539 [migration]

## Description

Has a migration

- Adds additional materials button.
- Adds prompt guidance for additional materials
- Refractors modify button to reduce duplication
- add e2e test

EDIT - Removed option for translating keywords new ticket [here ](https://www.notion.so/oaknationalacademy/Additional-materials-modifications-other-language-13526cc4e1b180d2afa3c91e56cfbd36)

## Acceptance Criteria

https://www.notion.so/oaknationalacademy/In-app-feedback-additional-materials-10526cc4e1b180dd8ea9cf8654ae099d

- [ ]  When there is no additional materials **section** at all, no change to the current working
- [ ]  When there are no additional materials, modify button is replaced with a ‘Add additional materials’ button.
- [ ]  5 additional material options to be included in dropdown option (see above for options)
- [ ]  User can select an option to add relevant additional material to their lesson
- [ ]  User can select other and provide details of an additional material they’d like to add
- [ ]  For the language option, Aila should follow up and ask which language to use
- [ ]  When there are additional materials, option copy to be modified to suggestion above.

## How to test

1. Go to https://oak-ai-lesson-assistant-ro5gl0ww4.vercel-preview.thenational.academy
Create lesson - when a lesson has 'none' for additional resources you should see 'add additional resource' button with different options. Modify should work as before.



## Checklist
